vendor/github.com/knative/pkg/test/request.go

```diff
@@ -34,6 +34,28 @@ func MatchesAny(_ *spoof.Response) (bool, error) {
 	return true, nil
 }
 
+// RetryingRouteUpdate modifies a ResponseChecker to retry response code
```

Comment:
I guess it's a question for the original pr, but it seems word `Route` here is excessive, we just do a generic retry?

Reply:
Well, this change hasn't been made yet.  I'd call it something else in `knative/pkg`